src/js/components/Button/stories/Normalized.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { User } from 'grommet-icons';
+import { Box, Button, grommet, Grommet, Text } from 'grommet';
+import { deepMerge } from 'grommet/utils';
+
+const customTheme = deepMerge(grommet, {
+  button: {
+    normalized: {
+      color: '#ff9999',
+      hover: {
+        border: {
+          color: '#99ccff',
+        },
+      },
+    },
+  },
+});
+
+const Normalized = () => (
+  <>
```

Comment:
I don't think you need a fragment.

Reply:
I used two `Grommet` components to be able to demonstrate with different themes, so I need the fragment to wrap around the two of them.

Reply:
Gotcha, I'll dive deeper into the implementation later on today.